src/Klsandbox/BillplzRoute/Services/BillplzResponseManager.php

```diff
@@ -80,17 +80,17 @@ public function createBill($data)
         return $return;
     }
 
-    public function createCollection()
```

Comment:
Delete the caller 
    public function getNewCollection()

Delete the view for that caller

Delete route entry for that caller